Makefile

```diff
@@ -15,7 +16,12 @@ spec-tmp.yaml:
 
 # YAML linting
 lint: spec-tmp.yaml
-	yamllint -c ./yamllint/config.yaml ./spec.yaml ./definitions.yaml ./parameters.yaml ./responses.yaml
+	docker run --rm -ti \
+		-v $(PWD):/workdir \
+		-w /workdir \
+		giantswarm/yamllint \
+		-c ./yamllint/config.yaml \
+		./spec.yaml ./definitions.yaml ./parameters.yaml ./responses.yaml
```

Comment:
This change is unrelated. It allows to run the linting without having `yamllint` available.